android/app/src/main/java/io/mosip/registration_client/api_services/MasterDataSyncApi.java

```diff
@@ -142,27 +149,28 @@ public MasterDataSyncApi(ClientCryptoManagerService clientCryptoManagerService,
         this.languageRepository = languageRepository;
         this.jobManagerService = jobManagerService;
         this.auditManagerService = auditManagerService;
+        this.masterDataService = masterDataService;
     }
 
 
-    private void syncPolicyKey(@NonNull MasterDataSyncPigeon.Result<MasterDataSyncPigeon.Sync> result) {
+    private void syncPolicyKey(@NonNull MasterDataSyncPigeon.Result<MasterDataSyncPigeon.Sync> result, @NonNull String APP_ID, @NonNull String REF_ID, @NonNull String SET_APP_ID, @NonNull String SET_REF_ID) {
```

Comment:
variable name within the function shouldn't be in this format
"REF_ID", it's only used for constants.

what does REF_ID and SET_APP_ID SET_REF_ID and SET_REF_ID mean here, this should be explained

Reply:
I have made changes